.devcontainer/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM containerbase/node:14.20.1@sha256:7772af27efa5703ee593b158e18c4160dff0d7d605d2e48e1d0b5ecdb467556d
+FROM containerbase/node:16.17.1
 
 USER root
 
```

Comment:
We may also need to edit the GitHub Action files, they still mention Node 14. For example:

https://github.com/renovatebot/renovate/blob/0ab15a4b214fb16f69f8c8ece9ae81b438a650f7/.github/workflows/build.yml#L27

https://github.com/renovatebot/renovate/blob/0ab15a4b214fb16f69f8c8ece9ae81b438a650f7/.github/workflows/build.yml#L41

https://github.com/renovatebot/renovate/blob/0ab15a4b214fb16f69f8c8ece9ae81b438a650f7/.github/workflows/build.yml#L32

Probably a good idea to search for `14` in the `.github` folder, and see what pops up. 😉

Reply:
will be done in a later pr